packages/paymaster/src/interfaces/IHybridPaymaster.ts

```diff
@@ -0,0 +1,20 @@
+import { UserOperation } from '@biconomy/core-types'
+import {
+  BiconomyTokenPaymasterFeeQuoteResponse,
+  BiconomyTokenPaymasterRequest,
+  FeeQuotesOrDataDto
+} from '../utils/Types'
+import { Transaction } from '@biconomy/core-types'
+import { Provider } from '@ethersproject/abstract-provider'
+import { IPaymaster } from './IPaymaster'
+
+export interface IHybridPaymaster extends IPaymaster {
+  createTokenApprovalRequest(
+    tokenPaymasterRequest: BiconomyTokenPaymasterRequest,
+    provider: Provider
+  ): Promise<Transaction>
+  getPaymasterFeeQuotesOrData(
+    userOp: Partial<UserOperation>,
+    paymasterServiceData: FeeQuotesOrDataDto
```

Comment:
change paymasterServiceData to feeQuoteInfo or feeQuoteOrData

Reply:
it could actually be just account information (for sponsorship mode) or feeQuote specific info like preferredToken
so really hard to decide the name and kept generic paymasterServiceData

Reply:
base on type, i guess using feeQuoteOrData would be better

Reply:
then i have to call it feeQuoteOrDataDto (basically camel case of dto name)
better is paymasterServiceData